src/editors/containers/ProblemEditor/components/EditProblemView/SettingsWidget/hooks.test.js

```diff
@@ -278,6 +282,7 @@ describe('Problem settings hooks', () => {
       expect(setBlockTitle).toHaveBeenCalledWith('Dropdown');
       expect(updateAnswer).toHaveBeenNthCalledWith(1, { ...answers[0], correct: false });
       expect(updateAnswer).toHaveBeenNthCalledWith(2, { ...answers[1], correct: false });
+      expect(updateAnswer).not.toHaveBeenNthCalledWith(3, { ...answers[2], correct: false });
```

Comment:
good catch, this will catch the bug if it occurs again.

nit: if we find a bug, we want to regression test it. That would ideally mean having a separate test that explains what the bug is as part of the test name, and in general a test should only test one thing. Tests should also not rely on implementation details, since that leads to false positives and false negatives. However, fixing that is probably a bit too big of a scope for this PR.